portfolio/src/main/webapp/discussion.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
frontend feature can be removed

Reply:
Even though the edit is on an HTML, i needed to make that edit to get the backend to load correctly

Reply:
I'm confused, does the Servlet not run successfully if there is no mainapp?

Anyway, have this in sync with other comments from other pull requests so that no merge is necessary.

Reply:
I just mean without the front end aspect, I would have difficulty seeing if it's working because I wouldn't be able to input in data.